README.adoc

```diff
@@ -117,17 +119,22 @@ The template interface describes the remote service that you want to access. The
 `src/main/java/io/openliberty/guides/inventory/client/SystemClient.java`
 ----
 SystemClient.java
-[source, Java, linenums, role='code_column']
+[source, Java, linenums, role='code_column hide_tags=copyright']
 ----
-include::finish/src/main/java/io/openliberty/guides/inventory/client/SystemClient.java[tags=**;!copyright]
+include::finish/src/main/java/io/openliberty/guides/inventory/client/SystemClient.java[]
 ----
 
-The MicroProfile Rest Client feature automatically builds and generates a client implementation based on what is defined in the [hotspot=17-22 file=2]`SystemClient` interface. Thus, you don't waste time on setting up the client and connecting with the remote service.
+The MicroProfile Rest Client feature automatically builds and generates a client implementation based on what is defined in the [hotspot=SystemClient file=2]`SystemClient` interface. Thus, you don't waste time on setting up the client and connecting with the remote service.
+
+Notice the [hotspot=SystemClient file=2]`SystemClient` interface inherits the [hotspot=AutoCloseable file=2]`AutoCloseable` interface.
+The client service will automatically close after its invocation, and the underlying resources are cleaned up.
```

Comment:
I don't like the wording here.  The way I read this is "the client instance is closed automatically after I call one method on it".  That is not correct.

Instead, it allows client interfaces to be closed by either the user explicitly calling the `close()` method or by using the client interface in a try-with-resources block - then the client is closed when that block has finished executing - for example:

```
MyClient client = RestClientBuilder.newBuilder().baseUri(someURI).build(MyClient.class);
try (MyClient c = client) {
    c.doSomething();
    ...
    c.doSomethingElse();
} // client is closed here
catch (Exception ex) {
...
```

So maybe this sentence should be phrased like:
This allows the user to explicitly close the client instance by invoking the `close()` method or to implicitly close the client instance using a try-with-resources block.  When the client instance is closed, all underlying resources associated with the client instance are cleaned up.

Reply:
Thank you. Should have tested it out. Fixed.